app/templates/core/subscribe-modal.jade

```diff
@@ -1,3 +1,17 @@
+mixin price(name, p)
+  - var origPrice = p.priceStringNoSymbol()
+  if origPrice % 1 === 0
+    - origPrice = Math.floor(origPrice);
```

Comment:
If `origPrice` is an integer, doesn't this do nothing? Maybe add a comment if I misunderstand.

Reply:
It's kinda worse, `origPrice` is a string of a float, which converts back to a number when you mod it.  Going to leave as is for now.